lib/application/presentation/detail_screen.dart

```diff
@@ -1,25 +1,294 @@
+                                      fontSize: 18,
+                                      fontWeight: FontWeight.w700),
+                            ),
+                          ))),
+                ])
+              ],
+            ),
+            Align(
+              alignment: Alignment.centerLeft,
+              child: Padding(
+                padding: const EdgeInsets.only(
+                    left: 40, right: 40, top: 10, bottom: 5),
+                child: Text("Speaker",
+                    textAlign: TextAlign.left,
+                    style: Theme.of(context).textTheme.labelMedium?.copyWith(
+                        fontFamily: 'Montserrat',
+                        fontWeight: FontWeight.w700,
+                        color: MyColors.colorFFFFFF,
+                        fontSize: 14)),
+              ),
+            ),
+            const Divider(
+              height: 1,
+              color: MyColors.color772DFF,
+              indent: 40,
+              endIndent: 40,
+              thickness: 1,
+            ),
+            ListView.builder(
+              shrinkWrap: true,
+              itemCount: state.speakers.length,
+              itemBuilder: (context, index) {
+                return listBuild(state.speakers[index]);
+              },
+            ),
+            Align(
+              alignment: Alignment.centerLeft,
+              child: Padding(
+                padding: const EdgeInsets.only(
+                    left: 40, right: 40, top: 20, bottom: 5),
+                child: Text("Description",
+                    overflow: TextOverflow.ellipsis,
+                    textAlign: TextAlign.left,
+                    style: Theme.of(context).textTheme.labelMedium?.copyWith(
+                        fontFamily: 'Montserrat',
+                        fontWeight: FontWeight.w700,
+                        fontSize: 14,
+                        color: MyColors.colorFFFFFF)),
+              ),
+            ),
+            const Divider(
+              height: 1,
+              color: MyColors.color772DFF,
+              indent: 40,
+              endIndent: 40,
+              thickness: 1,
+            ),
+            Expanded(
+                flex: 1,
+                child: Padding(
+                    padding:
+                        const EdgeInsets.only(left: 40, right: 40, top: 10),
+                    child: SingleChildScrollView(
+                      scrollDirection: Axis.vertical, //.horizontal
+                      child: Text(
+                        widget.scheduleItem.description,
+                        style: (Theme.of(context).textTheme.bodyLarge?.copyWith(
+                            fontFamily: 'Montserrat',
+                            fontWeight: FontWeight.w400,
+                            fontSize: 14)),
+                      ),
+                    )))
+          ],
+        );
+      },
+    ));
+  }
+
+  Widget listBuild(Speaker speaker) {
```

Comment:
Bilo bi dobro renamati ovu metodu, možda nešto tipa `speakerEntry`. Također, ovdje koristiš i Card i ListTile što je overkill. Ovdje je sasvim dovoljan jedan Row koji sadrži avatara i Column koji sadrži dva Text widgeta i to je to :)

Reply:
Koja je razlika onda izmedu Schedule i ovoga?, Dal se onda i tamo moglo samo Row koristit?, jer ovo bi isto trebala biti lista kao i tamo